test/SlowTests/Issues/RavenDB_8847.cs

```diff
@@ -77,29 +81,31 @@ public async Task Should_set_first_batch_timeout_of_newly_created_static_index()
 
                 database.IndexStore.StopIndexing();
 
-                var index = database.IndexStore.GetIndex(usersByname);
+                var index = database.IndexStore.GetIndex(usersByName);
 
-                using (var session = store.OpenSession())
+                using (var session = store.OpenAsyncSession())
                 {
-                    session.Query<User>(usersByname).Customize(x => x.WaitForNonStaleResults()).ToList();
+                    session.Query<User>(usersByName).Customize(x => x.WaitForNonStaleResults()).ToList();
```

Comment:
this does not throw? should be ToListAsync, right?

Reply:
done